builds/guardian/power-dragonhunter/index.md

```diff
@@ -242,6 +236,14 @@ It is not recommended to run <Trait name="Righthandstrength"/> unless you have m
 <GridItem sm="7">
 <Traits traits1Id="16" traits1="Radiance" traits1SelectedIds="574,565,579" traits2Id="42" traits2="Zeal" traits2SelectedIds="634,653,2017" traits3Id="27" traits3="Dragonhunter" traits3SelectedIds="1898,1835,1955"/>
 
+<Card title="Virtues">
+
+If you can reliably upkeep <Boon name="Aegis"/>, and you dont gain value from sharing <Skill name="Bane Signet"/> through <Trait name="Perfect Inscriptions"/>. You can drop the Radiance traitline for Virtues for more personal damage. Please note you will need to adjust your gear potentially to crit cap! Check the [gear optimizer](https://discretize.github.io/discretize-gear-optimizer) for more gear variants!
```

Comment:
Gear optimizer links should point to `optimizer.discretize.eu` now

Reply:
Fixed the links